fmath.hpp

```diff
@@ -102,7 +102,7 @@ template<class T, size_t N>
 size_t NumOfArray(const T (&)[N]) { return N; }
 
 /*
-	exp(88.722839f) = inf ; 0x42b17218
+	exp(88.722839f) = 1.65163596e+038f ; 0x42b17218
```

Comment:
It seems to me that previously, infinity was returned almost by accident (and as said, not for inputs such as `89.0f`). Now, as the input is explicitly clamped, this same value will be returned for all inputs that are greater than 88.

However, I did not try the Xbyak version. Guessing it would be good if the exact return values were equal...